src/rsz/README.md

```diff
@@ -115,15 +115,17 @@ Standard Parasitic Exchange Format.
 
 ```tcl
 estimate_parasitics
-    -placement|-global_routing
+    -placement|-global_routing|-detailed_routing
     [-spef_file spef_file]
+    [-ext_model_file]
```

Comment:
```suggestion
    [-ext_model_file model_file]
```

Reply:
how does this handle corners? they have separate model files.

Reply:
In my tests, I'm using only the public PDKs available in ORFS, which use only on model file and one corner for parasitics extraction.

How are you using multiple corners in your flow? Do you loop over each corner to extract the parasitics for them?

Reply:
Yes, we loop over them to generate a different spef file per corner and read them in per corner to be able to get timing across corners (it's clumsy, but it's the interface we have in OpenROAD at the moment). I think corner support is needed for this to be a complete/correct solution.

Reply:
If rcx was augmented to allow us to set the model files per corner that would also allow us to avoid adding this argument and just ask rcx to generate the parasitics

Reply:
Maybe this option can be updated to take multiple files, and them the implementation will perform parasitics extraction for each of them. I believe sta have the support to have the parasitics stored for multiple corners, so when running repair_timing it would look at all of them.

Reply:
@eder-matheus it seems to me like modifying RCX to store the model files would require a much less complex argument to this command. and yes, STA fully supports corners.